caab-service/src/main/java/uk/gov/laa/ccms/caab/api/entity/AuditTrail.java

```diff
@@ -36,22 +37,20 @@ public class AuditTrail {
    */
   @Column(name = "MODIFIED")
   @UpdateTimestamp
-  private Date modified;
+  private Date lastSaved;
```

Comment:
Why have we deviated from the column names?

Reply:
simplified some mapping issues i was having, can discuss if you want. 
No reason we cant change them as long as the column is set correctly?

Reply:
Just for ease of understanding really. 
But I've just checked and I don't think the column names conform to the Oracle WHO naming standards anyway.
https://docs.oracle.com/cd/E18727_01/doc.121/e12897/T302934T303920.htm